src/sonarwhal-theme/source/js/helpers/compare.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-env browser */
```

Comment:
All this `ESLint` configurations should go in the `.estlintrc` file in the `sonarwhal-theme/source/js` folder

Reply:
ok